packages/mobile/src/services/remote-config/remote-config-instance.ts

```diff
@@ -19,31 +19,43 @@ const { version: appVersion } = packageInfo
 const OPTIMIZELY_KEY = Config.OPTIMIZELY_KEY
 const DATA_FILE_URL = 'https://experiments.audius.co/datafiles/%s.json'
 
-/** Returns version in the form of `a1.1.33+codepush:v5` if Android, `i1.1.33+codepush:v5` if iOS. The `+codepush:*` portion is omitted if there is no CodePush update installed. */
-const getMobileClientVersion = async () => {
-  const baseVersion = `${Platform.OS === 'android' ? 'a' : 'i'}${
-    VersionNumber.appVersion
-  }`
-  let res = baseVersion // This is our default value if getting the CodePush update metadata fails for some reason, or if there is no CodePush update installed.
-  let codePushUpdateMetadata: LocalPackage | null
+/** Returns mobile platform (ios or android), mobile app version, and code push update number (if any) */
+const getMobileClientInfo = async () => {
+  const mobilePlatform = (Platform.OS === 'android' ? 'android' : 'ios') as
```

Comment:
can do "as const" i think here?

Reply:
just wondering if it could maybe even just be set to "Platform.OS"?

Reply:
yeah i think that'd be better, i orginally thought this would make it clear that we only expect android or ios

Reply:
yeah Os is android, i think the "saga" part is like Platform.constants.Model